src/tests/Microsoft.Agents.Builder.Testing/Adapters/MockUserTokenClient.cs

```diff
@@ -160,6 +160,16 @@ public Task<SignInResource> GetSignInResourceAsync(string connectionName, IActiv
             });
         }
 
+        public Task<TokenOrSignInResourceResponse> GetTokenOrSignInResourceAsync(string userId, string connectionName, string channelId, string code, string state, string finalRedirect, string fwdUrl, CancellationToken cancellationToken)
+        {
+            var tokenResponse = GetUserTokenAsync(userId, connectionName, channelId, code, cancellationToken).Result;
+            if (tokenResponse != null)
```

Comment:
```suggestion
            if (tokenResponse is not null)
```

https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/operators/patterns#constant-pattern